src/components/EditCollectiveForm.js

```diff
@@ -740,11 +744,50 @@ class EditCollectiveForm extends React.Component {
               {this.state.section === 'connected-accounts' && (
                 <EditConnectedAccounts collective={collective} connectedAccounts={collective.connectedAccounts} />
               )}
-              {archiveIsEnabled && this.state.section === 'advanced' && collective.type !== 'USER' && (
-                <ArchiveCollective collective={collective} />
-              )}
-              {deleteIsEnabled && collective.type !== 'EVENT' && this.state.section === 'advanced' && (
-                <DeleteCollective collective={collective} />
+              {this.state.section === 'advanced' && (
+                <Fragment>
+                  {collective.type === 'COLLECTIVE' && collective.host.hostCollective && (
+                    <Container display="flex" flexDirection="column" width={1} alignItems="flex-start">
+                      <H2>
+                        <FormattedMessage
+                          id="collective.balance.title"
+                          defaultMessage={'Empty balance of Collective'}
```

Comment:
Just out of curiosity: is there advantages to use `={'str'}` instead of `="str"`?